src/index.html

```diff
@@ -1,13 +1,309 @@
+<!-- #region Head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Nothing</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
+    <link rel="icon"
+          type="image/jpg"
+          href="images/favicon.jpg"
+    >
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+
+      <div class="container">
+        <div class="header__content">
+          <div class="top--bar">
+            <a href="#" class="top--bar__logo-link">
+              <img src="images/logo.png"
+               alt="NOTHING logo"
+               class="top--bar__logo">
+            </a>
+            <div class="top--bar__icons">
+              <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header_bottom">
+            <h1 class="header__title">
+              Bring joy back
+              <br>
+              to the everyday
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+
+      <div class="container">
+        <div class="top--bar menu__top">
+          <a href="#" class="top--bar__logo-link">
+            <img src="images/logo.png"
+              alt="NOTHING logo"
+              class="top--bar__logo"
+              >
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#products">Products</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#store">Store</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about us">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contact us">Contact Us</a>
+              </li>
+              <li>
+                <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
```

Comment:
Need to fix format phone only in href everywhere
```suggestion
                <a href="tel:+123455555555" class="menu__phone-number">
```